include/graphblas/base/vector.hpp

```diff
@@ -236,6 +236,14 @@ namespace grb {
 				(void) n;
 			}
 
+			/**
+			 * Creates a dense ALP/GraphBLAS vector. This constructor takes an initializer
```

Comment:
"This constructor [..]" and onwards should probably part of a `@param[in] vals` doxygen clause?